lib/open3.rb

```diff
@@ -482,44 +482,95 @@ class << self
     private :popen_run
   end
 
-  # Open3.capture3 captures the standard output and the standard error of a command.
+  # :call-seq:
+  #   Open3.capture3([env, ] command_line, options = {}) -> [stdout_s, stderr_s, status]
+  #   Open3.capture3([env, ] exe_path, *args, options = {}) -> [stdout_s, stderr_s, status]
   #
-  #   stdout_str, stderr_str, status = Open3.capture3([env,] cmd... [, opts])
+  # Basically a wrapper for Open3.popen3 that:
   #
-  # The arguments env, cmd and opts are passed to Open3.popen3 except
-  # <code>opts[:stdin_data]</code> and <code>opts[:binmode]</code>.  See Process.spawn.
+  # - Creates a child process, by calling Open3.popen3 with the given arguments
+  #   (except for certain entries in hash +options+; see below).
+  # - Returns as strings +stdout+ and +stderr+ the standard output
+  #   and standard error of the child process.
+  # - Returns as +status+ a <tt>Process::Status</tt> object
+  #   that represents the exit status of the child process.
   #
-  # If <code>opts[:stdin_data]</code> is specified, it is sent to the command's standard input.
+  # With no block given, returns the array <tt>[stdin_s, stdout_a, status]</tt>:
```

Comment:
I think this is confusing because it does not accept a block.

Reply:
Fixed.